aptos-move/framework/aptos-framework/sources/fungible_asset.move

```diff
@@ -207,6 +212,14 @@ module aptos_framework::fungible_asset {
         object::object_from_constructor_ref<Metadata>(constructor_ref)
     }
 
+    public entry fun update_uris(metadata_owner: &signer, metadata_addr: address, new_icon_uri: String, new_project_uri: String) acquires Metadata {
```

Comment:
would we want to mention that this has security implications where metadata is no longer guaranteed to be immutable?